ci/aws/aws-cluster.lokocfg.envsubst

```diff
@@ -11,6 +11,7 @@ cluster "aws" {
   asset_dir        = pathexpand(var.asset_dir)
   cluster_name     = "$CLUSTER_ID"
   controller_count = 2
```

Comment:
I thought we use DaemonSet with multiple controllers, so there should be no multiple API servers on a single machine, right?

Reply:
Yep that's correct.